featuretools/computational_backends/feature_tree.py

```diff
@@ -25,16 +25,21 @@ def __init__(self, entityset, features, ignored=None):
             self.ignored = set([])
         else:
             self.ignored = ignored
+
+        self.feature_hashes = set([f.hash() for f in features])
+
         all_features = {f.hash(): f for f in features}
         feature_deps = {}
         for f in features:
             deps = f.get_deep_dependencies(ignored=ignored)
             feature_deps[f.hash()] = deps
             for dep in deps:
                 all_features[dep.hash()] = dep
-                feature_deps[dep.hash()] = dep.get_deep_dependencies(ignored=ignored)
+                feature_deps[dep.hash()] = dep.get_deep_dependencies(
+                    ignored=ignored)
         self.all_features = list(all_features.values())
         self.feature_deps = feature_deps
+        self.feature_dependents = self._build_dependents()
```

Comment:
i think this is the inverse of `self.feature_deps`, but the variable name doesn't make it very clear since it is the same

Reply:
i didn't want to change feature_deps when I was first doing it for fear it was used elsewhere, but now that everythign works it would be better to name it feature_dependencies